### PR TITLE
fix building documentation

### PR DESCRIPTION
**Pull request by przemyslaw-szejna-sb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ yarn install
 cd better-docs
 yarn install
 
-DOCS=../adminjs/src/**/*,../docs-src/**/*,../adminjs-mongoose/**/*,../adminjs-hapijs/**/*,../adminjs-expressjs/**/*,../adminjs-sequelizejs/**/* gulp
+DOCS=../adminjs/src/**/*,../docs-src/**/*,../adminjs-mongoose/**/*,../adminjs-hapijs/**/*,../adminjs-expressjs/**/*,../adminjs-sequelizejs/**/*,../adminjs-prisma/**/* yarn  --cwd better-docs/ gulp
```
Comment (dziraf) [inline README.md:94]: Do we need `--cwd better-docs/ gulp` if previous commands tell you to `cd better-docs`?

File: jsdoc.json [hunk 2]
```diff
@@ -1,96 +1,100 @@
...
-              ]
-            }
+    "better-docs": {
+      "landing": "./docs-src/landing.html",
+      "logo": "images/logo-reg.svg",
+      "title": "AdminJS - Auto-generated admin panel for node.js with react",
+      "component": {
+        "wrapper": "./adminjs/docs/component.tsx"
+      },
+      "trackingCode": "<!-- Google Tag Manager --><script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-5L24P7Z');</script><!-- End Google Tag Manager -->",
```
Comment (dziraf) [inline jsdoc.json]: Can you add a new GTM:
```
<!-- Google Tag Manager -->
<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-K6R3HZ4');</script>
<!-- End Google Tag Manager -->
```

There's also a second part of GTM which should go into `body`:
```
<!-- Google Tag Manager (noscript) -->
<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-K6R3HZ4"
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<!-- End Google Tag Manager (noscript) -->
```

**Comment by dziraf** [comment]
Logo needs to be updated to this one: https://docs.adminjs.co/images/logo-adminjs.svg
Previous one is in `images/logo-reg.svg`

**Comment by dziraf** [comment]
Update base url to `index.html` from `docs.html`, i. e.:

```
             <div class="navigation">
                 <a
-                    href="docs.html"
+                    href="index.html"
                     class="link"
                 >
```

**Comment by dziraf** [comment]
Make sure demo link is correct:
```
[https://admin-bro-example-app-staging.herokuapp.com/admin/login](https://admin-bro-example-app-staging.herokuapp.com/admin/login)
```

`demo.adminjs.co` doesn't work

File: README.md (hunk 1 above)
Comment (przemyslaw-szejna-sb) [reply README.md:94]: not really, I'll remove that

**Comment by przemyslaw-szejna-sb** [comment]
> Update base url to `index.html` from `docs.html`, i. e.:
> 
> ```
>              <div class="navigation">
>                  <a
> -                    href="docs.html"
> +                    href="index.html"
>                      class="link"
>                  >
> ```

Fixed by removing `landing` property from jsdoc.json

File: jsdoc.json (hunk 2 above)
Comment (przemyslaw-szejna-sb) [reply jsdoc.json]: I added a new property bodyTrackingCode and put it within body tags

**Review by dziraf — APPROVED** [review]

**Pull request merged**